### ISSUE TITLE
Use of ffmpeg

### ISSUE THREAD

**Issue by Quaker762** [Palm-Studios/sh3redux#59]
SILENT HILL 3 has a few *.000 videos inside of /data/movie/.

I have code to convert these into `.mpg` (MPEG1) files, but it's a bit of chore to actually get these to play in SDL (get frame, convert to texture and draw). Should we use ffmpeg for this, as it already does all of the work decoding the output? There's a tutorial on doing this with SDL, but it involves .avi instead of .mpg, though I believe it is possible to convert.

**Comment by z33ky** [comment]
We should be able to sample the frames FFmpeg produces (see `sws_scale()`) and then just upload that data as an OpenGL texture.

**Comment by Quaker762** [comment]
Yeah this is what I was thinking. I'll give it a crack in a separate branch. Hopefully we can get a scene manager and two scenes (logos/video) up in the next week or two.
